docs/source/advanced/advanced-usages.rst

```diff
@@ -22,6 +22,52 @@ To be more specific, these dataloaders will do neighbor sampling regardless of a
 
 With DGL 1.0.4, ``fast_localuniform`` dataloader can speedup 2.4X over ``localuniform`` dataloader on training a 2 layer RGCN on MAG dataset on four g5.48x instances.
 
+Hard Negative sampling in Link Prediction Training
+-------------------------------------
+GraphStorm provides support for users to define hard negative nodes for a positive edge during Link Prediction Training.
+Currently, hard negative nodes can only be destination nodes.
+For example, given an edge (``src_pos``, ``dst_pos``) and hard negative nodes ``hard_0`` and ``hand_1``, GraphStorm will construct two hard negative edges, i.e., (``src_pos``, ``hard_0``) and (``src_pos``, ``hand_1``).
```

Comment:
i think it's better to not use hard negative nodes. I don't think the concept is well defined.